server/db/schema/clubs.schema.ts

```diff
@@ -43,6 +53,7 @@ export const clubs = pgTable('clubs', {
 export const clubsRelations = relations(clubs, ({ many, one }) => ({
   clubMembers: many(clubMembers),
   clubSocials: many(clubSocials),
+  clubEvents: many(events),
```

Comment:
```suggestion
  clubEvents: many(events),
  clubMembers: many(clubMembers),
  clubSocials: many(clubSocials),
```
This is alphabetically ordered